AppAuth-JS/src/node_app/index.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2017 Google Inc.
```

Comment:
Change the copyright in every occurrence?

Reply:
Google's lib fork. Did not change code in this file